src/components/JobDetails/JobDetails.jsx

```diff
@@ -12,22 +56,31 @@ function JobDetails(props) {
     const renderForm = () => {
         return (
             <form>
-                <Field label="Select the country">
-                    <SingleSelect disabled>
-                    </SingleSelect>
-                </Field>
-                <Field label="Name the catchment areas">
-                    <Input disabled/>
-                </Field>
-                <Field label="Select the facility level">
-                    <SingleSelect disabled>  
-                    </SingleSelect>
-                </Field>
-                <Field label="Select the groups">
-                    <MultiSelect disabled> 
-                    </MultiSelect>
-                </Field>
-            </form>
+            <Field label="Select the country" >
+                <SingleSelect selected={boundaries ? country : null} disabled>
+                    {boundaries && boundaries.map((bound, index) => {
+                        return <SingleSelectOption key={`boundary-${index}`} value={bound.id} label={`${bound.countryName}`}/>
+                    })}
+                </SingleSelect>
+            </Field>
+            <Field label="Name the catchment areas">
+                <Input value={name} disabled/>
+            </Field>
+            <Field label="Select the facility level">
+                <SingleSelect selected={levels ? selectedLevel : null} disabled>
+                    {levels && levels.map((level, index) => {
+                        return <SingleSelectOption key={index} label={level.name} value={level.id}/>
+                    })}
+                </SingleSelect>
+            </Field>
+            <Field label="Select the groups">
+                <MultiSelect selected={groups ? selectedGroup : []} disabled>
+                    {groups && groups.map((group, index) => {
+                        return <MultiSelectOption key={index} label={group.name} value={group.id}/>
+                    })}
+                </MultiSelect>
+            </Field>
+        </form>
```

Comment:
<img width="577" alt="Screen Shot 2022-05-02 at 10 09 14 AM" src="https://user-images.githubusercontent.com/13878539/166249128-d90b25ba-1497-42e6-b1e5-852a3fa96311.png">

The text looks slightly different for the different types of options (different levels of grey shading). It looks like this might just be because the different types of options have different color disabled values, but it might be nice for it to me more consistent if it's a simple change.